src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -198,8 +198,8 @@
   <data name="net_http_client_send_error" xml:space="preserve">
     <value>An error occurred while sending {0}. {1}</value>
   </data>
-  <data name="net_http_client_send_canceled" xml:space="preserve">
-    <value>Request for {0} was canceled.</value>
+  <data name="net_http_client_send_cancelled" xml:space="preserve">
+    <value>Request for {0} was cancelled.</value>
```

Comment:
This is not a misprint.  It was explicitly spelled that way (with one "L").  There are two accepted spellings of the word "canceled".  It turns out that much of the .NET Framework is inconsistent on this.